carbon-design-system/carbon-for-ibm-dotcom
# fix(feature-section): adjust grid alignment issues on certain breakpoints

### Related Ticket(s)

DDS Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7343
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2258

### Description
- This PR fixes some grid alignment issues that make it inconsistent with the design specs.

### Testing Instructions

![Screen Shot 2021-10-13 at 1 01 11 PM](https://user-images.githubusercontent.com/1815714/137179664-c10b61f3-ac81-4a7d-89ac-7cb6ea5be815.png)

- Visit the `Feature Section` layout Component - `/?path=/story/components-feature-section--default`.
- Use the Carbon Dev Tool extension and ensure that the main content container and card content are aligning to the grid defined in the design specs.
